polyfill/test/parseResults.js

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env node
+
+const { stdin, stdout, exit } = process;
+let testOutput = '';
+const now = Date.now();
+
+stdin.setEncoding('utf8');
+
+stdout.write('Starting Test262 tests.\n');
+
+let testCount = -1;
+let lastReportedCount = -1;
+
+function reportStatus() {
+  const secs = (Date.now() - now) / 1000;
+  stdout.write(`\r${testCount - 1} tests completed in ${secs} seconds. (${Math.round(testCount / secs)} tests/sec)   `);
+}
+
+stdin.on('data', function (chunk) {
+  const lines = chunk.split('\n').length;
+  testCount += lines;
+  if (testCount > lastReportedCount + 200) {
+    reportStatus();
+    lastReportedCount = testCount;
+  }
+  testOutput += chunk;
+});
+
+stdin.on('end', function () {
+  reportStatus();
```

Comment:
Not sure why you're calling this twice

Reply:
I wanted to show the total before and after the 1000+ tests listed. That said, in the latest update I just pushed, the repeating is not needed.